pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+# Overview:
+Give a brief overview of the issue you are solving. Succintly explain the GitHub issue you are addressin and the underlying problem of the ticket.
```

Comment:
typo: addressin

Reply:
And "Succintly"

Reply:
Thanks for the comments, fixed the typos.